SpokestackTests/NLUTensorflowTokenizerTest.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  NLUTensorflowTokenizerTest.swift
+//  SpokestackTests
+//
+//  Created by Noel Weichbrodt on 1/24/20.
+//  Copyright © 2020 Pylon AI, Inc. All rights reserved.
+//
+
+import XCTest
+import Spokestack
+
+class NLUTensorflowTokenizerTest: XCTestCase {
+    
+    func testWordpieceTokenize() {
+        let tokenizer = WordpieceTokenizer(try! SharedTestMocks.createEncodingsDictionary(SharedTestMocks.createVocabularyPath()))
```

Comment:
There's a lot of temp file creation/reading going on in this test. You should be able to simplify it with a `setUp` for the class.